packages/test/end-to-end-tests/src/test/container.spec.ts

```diff
@@ -17,15 +17,14 @@ import { MockDocumentDeltaConnection } from "@fluid-internal/test-loader-utils";
 import { LocalCodeLoader, TestObjectProvider } from "@fluidframework/test-utils";
 import { ensureFluidResolvedUrl } from "@fluidframework/driver-utils";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
-// eslint-disable-next-line import/no-extraneous-dependencies
-import { ITestDriver, LocalServerTestDriver } from "@fluidframework/test-drivers";
+import { LocalServerTestDriver } from "@fluidframework/test-drivers";
 import { createPrimedDataStoreFactory, createRuntimeFactory, TestDataObject } from "./compatUtils";
 
 const id = "fluid-test://localhost/containerTest";
 const testRequest: IRequest = { url: id };
 
 describe("Container", () => {
-    let driver: ITestDriver;
+    let driver: LocalServerTestDriver;
```

Comment:
Will we want to follow up here and run these tests against each different driver?  Worth opening an issue for?

Reply:
that's the next set of PR i have planned. just trying to put it in in smaller chunks